app/src/main/java/com/paocu/xviss/activities/adapter/TravelItemAdapter.java

```diff
@@ -48,13 +51,17 @@ public void onBindViewHolder(@NonNull TravelItemAdapter.ViewHolder holder, int p
         holder.deleteButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                System.out.println(travel);
+                System.out.println(position);
```

Comment:
Do not use console printing for debugging purposes. Set a proper logger instead.